zkg.meta

```diff
@@ -1,16 +1,8 @@
 [package]
 script_dir = scripts
-test_command = cd testing && btest -c btest.cfg
+test_command = make test
 summary = A Zeek package to detect CVE-2022-26937, a Windows NFS vulnerabilty.
 description = A Zeek package to detect CVE-2022-26937, a Windows NFS vulnerabilty.
+corelight_name = CVE-2022-26937
```

Comment:
probably remove corelight-specific things?

Reply:
Thanks @ynadji. Fixed it.